packages/cli/src/scripts/verify.ts

```diff
@@ -2,7 +2,9 @@ import ControllerFor from '../models/network/ControllerFor';
 import { VerifyParams } from './interfaces';
 
 export default async function verify(contractAlias, { network = 'mainnet', txParams = {}, networkFile, optimizer = false, optimizerRuns = 200, remote = 'etherchain', apiKey }: VerifyParams): Promise<void | never> {
-  if (remote === 'etherscan' && !apiKey) throw new Error('Etherscan API key not specified. To get one, follow this link: https://etherscan.io/myapikey');
+  if (remote === 'etherscan' && !apiKey) {
+    throw new Error('Etherscan API key not specified. To get one, follow this link: https://etherscancom.freshdesk.com/support/solutions/articles/35000022163-i-need-an-api-key');
```

Comment:
Same as above, revert or change the etherscan API url to https://etherscan.io/myapikey